mcda/utils/utils_for_main.py

```diff
@@ -1,20 +1,35 @@
+
+import os
 import argparse
 import json
 import pickle
 import random
-from typing import Union, Any, List
+import logging
+import sys
+from typing import Union, Any, List, Tuple
 from typing import Optional
 
+import numpy as np
 import pandas as pd
+from datetime import datetime
 from sklearn import preprocessing
 from sklearn.preprocessing import MinMaxScaler
 
+import mcda.utils.utils_for_parallelization as utils_for_parallelization
+import mcda.utils.utils_for_plotting as utils_for_plotting
 from mcda.configuration.config import Config
-from mcda.utils.utils_for_parallelization import *
-from mcda.utils.utils_for_plotting import *
 from mcda.mcda_without_robustness import MCDAWithoutRobustness
 from mcda.mcda_with_robustness import MCDAWithRobustness
 
+DEFAULT_INPUT_DIRECTORY_PATH = './input_files'  # present in the root directory of ProMCDA
+DEFAULT_OUTPUT_DIRECTORY_PATH = './output_files' # present in the root directory of ProMCDA
+
+custom_input_path = os.environ.get('PROMCDA_INPUT_DIRECTORY_PATH')  # check if an environmental variable is set
```

Comment:
This will work as a constant which will not be changed during the application run. Do you think capitalising the variable will be useful here as per the standards?

Applicable for all four of the variables related to path.

Reply:
Yes, but only for `custom_input_path` and `custom_output_path`? While `input_directory_path` and `output_directory_path` are variables (even though they - for now - don't change later on)?

Reply:
Perfect thanks. 
If you expect them to change later then leave them in small case.